Caboodle/Caboodle.csproj

```diff
@@ -39,6 +39,10 @@
     </UpgradeBackupLocation>
     <OldToolsVersion>2.0</OldToolsVersion>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard1.0|AnyCPU'">
+    <DebugType>pdbonly</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+  </PropertyGroup>

```

Comment:
Is this needed? It looks suspicious.

Reply:
Yeah, soooooo this was an issue because I couldn't debug the samples without it! I think it could be simplified down.